frontend/src/instance-views/ComparisonView.svelte

```diff
@@ -124,58 +180,116 @@
 
 		await tick();
 
-		[$model, ...$comparisonModels].forEach((mod) => {
+		[$model, $comparisonModel].forEach((mod) => {
 			let obj = $status.completeColumns.find((c) => {
 				return c.columnType === ZenoColumnType.OUTPUT && c.model === mod;
 			});
 			let modelColumn = obj ? columnHash(obj) : "";
-
-			tables[mod].forEach((_, i) => {
-				let div = viewDivs[mod][i];
-				if (div) {
-					viewFunction(
-						div,
-						viewOptions,
-						tables[mod][i],
-						modelColumn,
-						columnHash($settings.labelColumn),
-						columnHash($settings.dataColumn),
-						idHash
-					);
-				}
-			});
+			if (tables[mod] && viewDivs[mod]) {
+				tables[mod].forEach((_, i) => {
+					let div = viewDivs[mod][i];
+					if (div) {
+						viewFunction(
+							div,
+							viewOptions,
+							tables[mod][i],
+							modelColumn,
+							columnHash($settings.labelColumn),
+							columnHash($settings.dataColumn),
+							idHash
+						);
+					}
+				});
+			}
 		});
 	}
 </script>
 
 <Svelecte
-	options={$models.filter((m) => m !== $model)}
-	labelAsValue={true}
-	closeAfterSelect={true}
-	bind:value={$comparisonModels}
-	multiple
-	placeholder="Select models to compare" />
+	style="padding-top: 5px;padding-bottom: 5px"
+	value={selectColumn}
+	placeholder={"Column"}
+	valueAsObject
+	valueField={"name"}
+	{options}
+	on:change={(e) => {
+		if (e.detail !== selectColumn) {
+			selectColumn = e.detail;
+			// reset tables data to prevent rerender the existing(non-updated) data
+			tables[$model] = [];
+			tables[$comparisonModel] = [];
+			compareSort.set([undefined, true]);
+		}
+	}} />
 
 <div class="table-container" bind:this={instanceContainer}>
-	{#if tables[$model]}
+	{#if tables[$model] && tables[$comparisonModel]}
 		<table>
```

Comment:
Can we top-align the instance views in the table? idk if this can be done outside of the instance view.

Reply:
Can we also top-align the table header